test/fluree/db/policy/transact_test.clj

```diff
@@ -98,8 +98,7 @@
                      :schema/name "Alice",
                      :schema/email "alice@foo.bar",
                      :schema/birthDate "2022-08-17",
-                     :schema/ssn "111-11-1111",
-                     :ex/location {:id nil}}]
```

Comment:
Do we not return ref predicates in a select * now?

Reply:
This was a bug in main. That reference is an id set to nil. It should have never been returned in the first place.